### ISSUE TITLE
ASM: 'network traffic-manager profile' inconsistency in Status

### ISSUE THREAD

**Issue by HimanshuBri** [MSOpenTech/azure-xplat-cli#295]
Have attached screenshot for the same. Status is Disable while creating, but it becomes Enable while performing show command.
![image](https://cloud.githubusercontent.com/assets/10084788/8055613/73fdde38-0ebe-11e5-91ae-8c473dbf96eb.png)

**Comment by roman-boltunov** [comment]
Hi @HimanshuBri it takes a little time for traffic manager to change its state.

**Comment by HimanshuBri** [comment]
@roman-boltunov. That means it's happening at server end.

**Comment by roman-boltunov** [comment]
@HimanshuBri yes, we might want to mark it as external. Adding @anuchandy. Can you please clarify if it's ok to leave it as is, or should we report this issue and ask for some 'intermediate' state like 'updating' or 'enabling'?

**Comment by anuchandy** [comment]
This is external. We will report this to TM team.

**Comment by romangromov** [comment]
UPD: create is async opertation. so it's expected behavior
